### PR TITLE
Add details on using tasks from bundles

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Tasks are now published nightly to bundles.
Document how to use tasks from bundles.

Signed-off-by: Andrea Frittoli <andrea.frittoli@gmail.com>

# Submitter Checklist

Docs only! N/A: 

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

/kind documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,31 @@ status:
 ...
 ```
 
+### Using `Task`s through Bundles
+
+[Tekton Bundles](https://tekton.dev/docs/pipelines/pipelines/#tekton-bundles) are an alpha feature of Tekton pipelines that allows storing `Tasks` as bundles in a container registry, instead of resources in etcd in a Kubernetes cluster.
```
Comment (bittrance) [inline README.md:149]: Nitpick "... instead of as **custom** resources" perhpas?

**Review by bittrance — COMMENTED** [review]
Nice!

Would it make sense to include a link to https://console.cloud.google.com/gcr/images/tekton-releases/GLOBAL/catalog/upstream?gcrImageListsize=100 ?

**Review by bittrance — APPROVED** [review]

**Review by vdemeester — APPROVED** [review]

**Comment by afrittoli** [comment]
> Nice!
> 
> Would it make sense to include a link to https://console.cloud.google.com/gcr/images/tekton-releases/GLOBAL/catalog/upstream?gcrImageListsize=100 ?

Link added :)

**Review by vinamra28 — APPROVED** [review]
/lgtm
Thanks

**Pull request merged**